packages/aws-event-stream/src/aws_event_stream/aio/__init__.py

```diff
@@ -50,7 +48,7 @@ def __init__(
 
     async def send(self, event: E) -> None:
         if self._closed:
-            raise IOError("Attempted to write to closed stream.")
```

Comment:
For reviewers also wondering, this is related to: https://docs.astral.sh/ruff/rules/os-error-alias/